cluster-sync/sync.sh

```diff
@@ -25,10 +25,19 @@ if [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
   _kubectl wait deploy/imageio-deployment -n cdi --for=condition=Available --timeout=480s
 fi
 
-# TODO: fetch it dynamically
-export OPERATOR_VERSION=0.0.1
-
+# Build the operator:
+export RELEASE_VERSION=v0.15.2
+export GO111MODULE=on
+export OPERATOR_NAMESPACE=kubevirt
+export OPERATOR_VERSION=latest
+VERSION=${OPERATOR_VERSION} TARGET_NAMESPACE=${OPERATOR_NAMESPACE} make gen-manifests
+curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu && sudo mkdir -p /usr/local/bin/ && sudo cp operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu /usr/local/bin/operator-sdk && rm operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+operator-sdk --verbose build quay.io/kubevirt/vm-import-opreator:${OPERATOR_VERSION}
```

Comment:
I think we need to replace this with a verb in Makefile since it already fetches the operator-sdk and prepare its binary.
It will reduce the need to maintain the version of the operator-sdk outside of go.mod
https://github.com/kubevirt/cluster-network-addons-operator/blob/master/Makefile#L109

I'd also suggest to separate cluster-up / cluster-sync and operator-install into a separate step/file, i.e.
https://github.com/kubevirt/cluster-network-addons-operator/blob/master/cluster/operator-install.sh

So we'll get a clear view of the building blocks for test env, and all can be driven from the Makefile

Reply:
That's nice repo with nice examples, thanks for pointing it out, I wasn't aware of that.